tests/test_run_inputs.py

```diff
@@ -0,0 +1,129 @@
+from typing import Optional
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.remote.webdriver import WebDriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import (
+    ElementNotVisibleException,
+    ElementNotSelectableException,
+    NoSuchElementException,
+)
+from selenium.webdriver import ActionChains
+
+web = "https://machinist.flapflap.io/"
+run_inputs_path = "#__next > main > div.MenuBar__Root-sc-1th5wld-0.jDqDxJ.App__FloatingMenuBar-gv0e21-2.eZsxUV > ul > li:nth-child(4)"
+
+
+def test_run_input_header(driver: WebDriver):
+
+    driver.get(web)
+    driver.find_element(By.CSS_SELECTOR, run_inputs_path).click()
+
+    wait = WebDriverWait(
+        driver,
+        10,
+        poll_frequency=1,
+        ignored_exceptions=[
+            ElementNotVisibleException,
+            ElementNotSelectableException,
+            NoSuchElementException,
+        ],
+    )
+
+    def element_is_visible_and_has_text(driver: WebDriver) -> Optional[str]:
+        ele = driver.find_element(By.XPATH, '//*[@id="rcDialogTitle0"]')
+        if ele.is_displayed and ele.text != "":
+            return ele.text
+        else:
+            return None
+
+    element_inspect = wait.until(element_is_visible_and_has_text)
```

Comment:
Can you choose a more meaningful name for this variable? Like `text` or something.